### PR TITLE
Added UserRepository class configuration parameter

### PR DESCRIPTION
**Pull request by birkof**
- [x] When using different directory architecture you'll need to specify UserRepository class
```yaml
php_rbac:
    user_repository_class: Acme\Repository\UserRepository #(default to: App\Repository\UserRepository)
```

- [x] Enhanced CLI commands with more verbose output
- [x] Force use symfony packages for security, twig and orm
- [x] Lock libraries to current versions

### CODE REVIEW

**Comment by birkof** [comment]
@Olivier127 I need your input on this one too?

**Comment by Olivier127** [comment]
Hmm !! That's a good idea !! But maybe using UserRoleInterface would be more appropriate to further generalize.

File: src/Command/RbacAssignUserRoleCommand.php [hunk 1]
```diff
@@ -23,10 +18,8 @@
 class RbacAssignUserRoleCommand extends Command
 {
     public function __construct(
-        private PermissionRepository $permissionRepository,
         private RoleRepository $roleRepository,
-        private RoleManager $roleManager,
-        private UserRepository $userRepository
+        private $userRepository,
```
Comment (Olivier127) [inline src/Command/RbacAssignUserRoleCommand.php:22]: private ServiceEntityRepository $userRepository

**Comment by birkof** [comment]
> Hmm !! That's a good idea !! But maybe using UserRoleInterface would be more appropriate to further generalize. As I did with php_rbac.resolve_target_entities.permission and php_rbac.resolve_target_entities.role,

Are you saying to have another key like `resolve_target_repositories` where to define repositories?

**Comment by Olivier127** [comment]
no need to use resolve_target_repositories. All repositories inherits ServiceRepository. I'm just saying to use resolve_target_entities pour User entity.

**Comment by birkof** [comment]
Yea, but in that case we'll need to get the repository (used in `PhpRbacBundle\Command\RbacAssignUserRoleCommand`) from User entity. That means we'll need to extract it from EntityManager, de default one, or include another parameter in configuration to specify the current EntityManager.

**Comment by birkof** [comment]
@Olivier127 I've forced User entity in configuration and switched in `RbacAssignUserRoleCommand` from UserRepository to User entity & EntityManager. WDYT?

**Comment by Olivier127** [comment]
Great !

**Comment by birkof** [comment]
@Olivier127 pls bump minor version number too 🕺🏼

**Comment by Olivier127** [comment]
done :)

**Pull request merged**